save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/ExecutionView.kt

```diff
@@ -136,6 +136,7 @@ class ExecutionView : RComponent<ExecutionProps, ExecutionState>() {
                                 undefined
                             )
                         }.invokeOnCompletion {
+                            window.location.reload()
                             window.alert("Rerun request successfully submitted")
```

Comment:
I think this should be the other way around: at first user sees an alert, and once alert is dismissed, only then is the page reloaded

Reply:
Agree. Fixed